feature-group-definitions/select-show-picker.yml

```diff
@@ -0,0 +1,4 @@
+name: select.showPicker()
```

Comment:
For completeness:

- `HTMLSelectElement showPicker()`
- `<input> showPicker()`
- `showPicker() for HTMLSelectElement`